lib/sidekiq/cli.rb

```diff
@@ -269,6 +269,19 @@ def validate!
         logger.info @parser
         die(1)
       end
+
+      errors = []
```

Comment:
As a developer, if you see one error for your config values, you can pretty safely extrapolate that similar values will also be invalid.  I prefer the simpler impl in this case.